app/src/main/java/com/mk/tasky/agenda/data/mapper/PhotoMapper.kt

```diff
@@ -0,0 +1,21 @@
+package com.mk.tasky.agenda.data.mapper
+
+import com.mk.tasky.agenda.data.local.entity.PhotoEntity
+import com.mk.tasky.agenda.domain.model.AgendaPhoto
+
+fun AgendaPhoto.toEntity(eventId: String): PhotoEntity {
+    val key = if (this is AgendaPhoto.Local) null else (this as AgendaPhoto.Remote).key
```

Comment:
As per requirements, local photos don't need to be saved in the DB since they can't be edited in offline mode. If you still want to implement this, feel free to do so (you already advanced quite far with this project 👍 )